### ISSUE TITLE
[Expert] Agronomic recipe not found

### ISSUE THREAD

**Issue by TheMightyMcGrew** [EnigmaticaModpacks/Enigmatica6#5097]
### Modpack Version

1.4.1

### Describe your issue.

Expert recipe for argonomic sourcelink not recognized, responds with no recipe found. Game mode is in expert, attempted both brass and gold variants. PhytoGrow and arcane gold inlays were used, conduit placed and no recipe. Reloaded KubeJS scripts as well. Cycled in and out of expert mode to jog it, nothing seems to have worked. 

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by Quezler** [comment]
could you share a screenshot of the setup and the surrounding area?

**Comment by Quezler** [comment]
ideally with all the items on it, and you wouldn't happen to have any other pedestals nearby?

also, if you have discord & are in the enigmatica one, head to #support so we don't clutter github during debugging 👀

**Comment by TheMightyMcGrew** [comment]
I was there before coming here, to make sure as not to report a bug that wasnt in fact a bug :/ To answer your question no i dont have any anywhere else, I also removed my source jar, my crystallizer, everything related to ars nouveau. I broke and replaced them, made them the same level. I did however go into a seperate creative world and try to replicate. In there it worked. A redstone-ready world. I dont know whats caused this, since i downloaded the pack its been expert mode, and i havent had to do any creative fixing, or commands

**Comment by Darkere** [comment]
Doesn't this Ars crafting setup require a base for the centerpiece?

**Comment by TheMightyMcGrew** [comment]
the core is underneath, other recipes have worked up until this point, and continue to work. The only issue atm is the agronomic sourcelink

**Comment by TheMightyMcGrew** [comment]
Ive found the issue, its related to claimed or loaded chunks, credit to Captain for helping deduce this

**Comment by MuteTiefling** [comment]
Cannot reproduce this. Crafted without issue in claimed chunks.

Are you certain that there were no other pedestals nearby? I was just tearing my hair out last night with a setup having the exact same issue until my brain lit up and realized my source gen (mycelial with food going into a pedestal) was too close to the enchanter.

**Comment by TheMightyMcGrew** [comment]
Well im a beginning player to this, so i didnt really have a lot of Ars Nouveau stuff anyhow, I had exactly what it took to make 1 operating apparatus, a crystallizer, and a source jar that was empty. As soon as someone in the discord suggested trying to turn off claimed chunks it worked immediately. Im wondering if its not just a slightly bugged world or something. I can try to reproduce as well later on. In the same world and in a different one, but itll have to be later on.

**Comment by Amadeus99** [comment]
I also had this problem.  I had the chunk claimed and forced loaded.  I had no other ars nouveau items placed in the world.  The enchanting apparatus activated and did its animation and used up all the ingrediants but nothing was returned.  After reading this, I unclaimed the chunk and then it worked fine.  I then reclaimed and force loaded the chunk but now it works every time.  I did not try a second time before unclaiming the chunk so I don't know if it was just something with the first time using the apparatus or what.  Not sure if this is helpful but I thought I would give what info I had.
